bot/bot.js

```diff
@@ -224,36 +233,72 @@ client.on('interactionCreate', async interaction => {
     }
 });
 
+
+
 // create a on message event
 client.on('messageCreate', message => {
+    var botMentioned = scanForKeyword(message.content, "chappie");
+
+    if (!botMentioned) {
+        logger.info("false");
+        return;
+    } else {
+        logger.debug("true");
+    }
+
     if (!ai_enabled) {
+        logger.info("AI is not enabled");
         return;
+    } else {
+        logger.debug("AI is enabled");
     }
+    
+
+    // Message content
+    const content = message.content
+
+    // Message author
+    const author = message.author.global_name
```

Comment:
This variable is optional, per docs, and should be handled.
```suggestion
    var author = message.author.global_name
    if (author == undefined) {
        author = message.author.username // as a fallback
    }
```
}

Reply:
Okay, it has been changed